css/landing.css

```diff
@@ -1,6 +1,7 @@
 * {
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
 }
 .container {
```

Comment:
Add the background color to the body, not just the container

![image](https://user-images.githubusercontent.com/78752405/144186089-ddef9c2e-f12a-4fce-8750-aae8df276bab.png)